pkgs/development/tools/mold/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  autoPatchelfHook,
+  cmake,
+  tbb,
+  llvmPackages_latest,
+  xxHash,
+  zlib,
+  openssl,
+}:
```

Comment:
```suggestion
{ stdenv
, fetchFromGitHub
, lib
, autoPatchelfHook
, cmake
, tbb
, llvmPackages_latest
, xxHash
, zlib
, openssl
}:
```

Reply:
As long as we don't have an official style guide that specifies the preferred formatting in this situation, I call this opinionated.

Reply:
That is not opinionated, this is the default style for nixpkgs.